Src/WitsmlExplorer.Frontend/components/StyledComponents/Buttons.tsx

```diff
@@ -0,0 +1,37 @@
+import { Button } from "@equinor/eds-core-react";
+import styled from "styled-components";
+import { Colors, colors } from "styles/Colors";
+
+// A styled button to get the correct color when using the ghost or outlined variants (no fill)
+export const StyledGhostButton = styled(Button)<{ colors: Colors }>`
+  white-space: nowrap;
+  color: ${(props) => props.colors.infographic.primaryMossGreen};
+`;
+
+export const OutlinedIconButton = styled(Button)`
```

Comment:
I think we can remove this one, as having several types of icon buttons in the application is not necessary.